sklearn/linear_model/tests/test_logistic.py

```diff
@@ -541,6 +541,46 @@ def test_logistic_regressioncv_class_weights():
     assert_array_almost_equal(clf_lib.coef_, clf_lbf.coef_, decimal=4)
 
 
+def test_logistic_regressioncv_sample_weights():
```

Comment:
I think I agree that if the code splits in the future it is best to have the tests for both. I will simply duplicate them for `LogisticRegression` now. I will also duplicate the tests in `test_logistic_regressioncv_class_weights` for `LogisticRegression`.